packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.test.tsx

```diff
@@ -146,6 +148,21 @@ describe('routes/ProductV2/SubscriptionCreate', () => {
       queryAllByText('30-day money-back guarantee')[0]
     ).toBeInTheDocument();
     expect(queryByText('Billing Information')).toBeInTheDocument();
+    expect(
+      document.getElementById('paypal-button-container')
```

Comment:
I didn’t know how else to get the div#paypal-button-container element from the page, so I just used `document.getElementById`. Open to better approaches.

Reply:
We use a data-testid attr.  See line 145 above.

Reply:
Thanks; I do ultimately need an `id` attribute to render the PayPal button using the PayPal page script (#7097 ), so I think I'll leave this here for now. I can always update the test with the final UI components.